API/DataModel/Entities/Order.cs

```diff
@@ -0,0 +1,36 @@
+﻿using API.DataModel.Entities.AspNetIdentity;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.DataModel.Entities
+{
+    public class Order
+    {
+        [Key]
```

Comment:
Jest jakiś szczególny powód dlaczego dałeś te atrybuty? Entity framework po nazwie ID sam sie domyśli że to klucz główny i jest auto generowany

Reply:
Racja, [Key] działa, gdy nazwa klucza nie jest standardowa.